compose.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  gasp-avs:
+    image: gaspxyz/gasp-avs:7387cc389edd5792421456e72c2096b1712d83b5
+    restart: unless-stopped
+    env_file: .env
+    volumes:
+      - ./data:/data
+      - ${ECDSA_KEY_FILE_HOST:-./operator_keys/ecdsa.key.json}:/app/operator_keys/ecdsa_key.json:readonly
+      - ${BLS_KEY_FILE_HOST:-./operator_keys/bls.key.json}:/app/operator_keys/bls_key.json:readonly
```

Comment:
these are no longer in .env, we can remove the volumes
OP will get the keys from the JSONs